airflow/dag_processing/bundles/manager.py

```diff
@@ -30,56 +31,59 @@
 
     from airflow.dag_processing.bundles.base import BaseDagBundle
 
+_bundle_config = {}
+
 
 class DagBundlesManager(LoggingMixin):
     """Manager for DAG bundles."""
 
-    @property
-    def bundle_configs(self) -> dict[str, dict]:
-        """Get all DAG bundle configurations."""
-        configured_bundles = conf.getsection("dag_bundles")
-
-        if not configured_bundles:
-            return {}
-
-        # If dags_folder is empty string, we remove it. This allows the default dags_folder bundle to be disabled.
-        if not configured_bundles["dags_folder"]:
-            del configured_bundles["dags_folder"]
+    def parse_config(self) -> None:
+        """
+        Get all DAG bundle configurations and store in module variable.
 
-        dict_bundles: dict[str, dict] = {}
-        for key in configured_bundles.keys():
-            config = conf.getjson("dag_bundles", key)
-            if not isinstance(config, dict):
-                raise AirflowConfigException(f"Bundle config for {key} is not a dict: {config}")
-            dict_bundles[key] = config
+        If a bundle class for a given name has already been imported, it will not be imported again.
+        """
+        configured_bundles = conf.getjson("dag_bundles", "backends")
 
-        return dict_bundles
+        if not configured_bundles:
+            return
+
+        if not isinstance(configured_bundles, list):
+            raise AirflowConfigException(
+                "Bundle config is not a list. Check config value"
+                " for section `dag_bundles` and key `backends`."
+            )
+        seen = set()
+        for cfg in configured_bundles:
+            name = cfg["name"]
+            if name in seen:
+                raise ValueError(f"Dag bundle {name} is configured twice.")
+            seen.add(name)
+            if name in _bundle_config:
+                continue
+            class_ = import_string(cfg["classpath"])
+            kwargs = cfg["kwargs"]
+            _bundle_config[name] = (class_, kwargs)
+
+        # remove obsolete bundle configs
```

Comment:
Not sure we should expect the config to change while a component is up. We can probably simply short circuit out if `_bundle_config` isn't empty.

Reply:
> We can probably simply short circuit out if _bundle_config isn't empty 

did not understand